### DISCUSSION TITLE
Welcome to PhonePi_SampleServer Discussions!

### DISCUSSION THREAD

**Discussion by priyankark** [priyankark/PhonePi_SampleServer#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by saleeta** [comment]
What is the sample rate? and how does the time field affect the data?
![image](https://user-images.githubusercontent.com/44166793/112984235-15a26780-9178-11eb-91fe-7657c58133e0.png)

**Reply by priyankark** [reply]
Hi Saleeta, apologies for the confusion with respect to the Time field.  The time field represents the time interval between streaming  any two data points. In a way its a way of setting the sampling rate.  Like if you set it to 5000, its going to send the sensor info every 5 seconds.
